src/main/java/org/fundacionjala/coding/escarleth/CesarEncryption.java

```diff
@@ -0,0 +1,50 @@
+package org.fundacionjala.coding.escarleth;
+
+/**
+ * Powered by Escarleth Ledezma Q.
+ * CifradoCesar is a class that contains methods to encode,decode,vigenere encripton.
+ */
+public class CesarEncryption {
```

Comment:
It is missing the vigenere cipher
also you should implement the encode and decode.